libraries/wasm-jit/Source/WAST/ParseNumbers.cpp

```diff
@@ -239,7 +239,7 @@ bool tryParseInt(ParseState& state,UnsignedInt& outUnsignedInt,I64 minSignedValu
 		return false;
 	};
 
-	outUnsignedInt = isNegative ? UnsignedInt(-I64(u64)) : UnsignedInt(u64);
+	outUnsignedInt = isNegative ? -UnsignedInt(I64(u64)) : UnsignedInt(u64);
```

Comment:
Do we still need the `I64` cast anymore?

Reply:
it does seem completely redundant now, and everything still passes with it removed..

Reply:
I was a little hesitant because this case is a little unique due to the u64->u32 truncation and wasn't sure if `-UnsignedInt(u64)` vs `UnsignedInt(-u64)` mattered.., but it seems equivalent.